packages/root-config/src/microfrontend-layout.html

```diff
@@ -54,6 +52,11 @@
           <application name="@jembi/dashboard-app"></application>
         </div>
       </route>
+      <route path="#!/users">
+        <div style="flex-grow: 1">
+          <application name="@jembi/users-app"></application>
+        </div>
+      </route>
     </div>
     <application name="@jembi/footer-app"></application>
```

Comment:
The footer should come below all other apps. The portal apps are inject using the `openhim-console-ems-container` id